stubs/python-dateutil/dateutil/parser/__init__.pyi

```diff
@@ -35,13 +37,30 @@ class parser:
         timestr: _FileOrStr,
         default: datetime | None = ...,
         ignoretz: bool = ...,
-        tzinfos: Mapping[str, tzinfo] | None = ...,
+        tzinfos: _TzInfo | None = ...,
+        *,
+        dayfirst: bool | None = ...,
+        yearfirst: bool | None = ...,
+        fuzzy: bool = ...,
+        fuzzy_with_tokens: bool = ...,
         **kwargs: Any,
```

Comment:
```suggestion
```
We can remove this because we've covered all `**kwargs` that it accepts, and any other kwargs will error out.